app/main.py

```diff
@@ -1,13 +1,25 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [format_linter_error(err) for err in errors],
+        "path": file_path,
+        "status": "failed"
+        if [format_linter_error(err) for err in errors]
```

Comment:
no need to iterate through errors for second time,f.e. u can check the len of errors list

Reply:
don't understand how to take this key value to check len without dict name